src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = i < figures.length / 2 ? figureSupplier.getRandomFigure() : figureSupplier
+                    .getDefaultFigure();

```

Comment:
Ternary operator is rather hard construction for fast reading so better to format it properly
```suggestion
            figures[i] = i < figures.length / 2 
                    ? figureSupplier.getRandomFigure() 
                    : figureSupplier.getDefaultFigure();

```